xml/System.Runtime.CompilerServices/SkipLocalsInitAttribute.xml

```diff
@@ -19,8 +19,16 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Used to indicate to the compiler that the .locals init flag should not be set in nested method headers when emitting to metadata.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This attribute is unsafe because it may reveal uninitialized memory to the application in certain instances (e.g., reading from uninitialized stackalloc'd memory). If applied to a method directly, the attribute applies to that method and all nested functions (lambdas, local functions) below it. If applied to a type or module, it applies to all methods nested inside. This attribute is intentionally not permitted on assemblies. Use at the module level instead to apply to multiple type declarations.
```

Comment:
```suggestion
This attribute is unsafe, because it may reveal uninitialized memory to the application in certain instances (for example, reading from uninitialized stack-allocated memory). If applied to a method directly, the attribute applies to that method and all its nested functions, including lambdas and local functions. If applied to a type or module, it applies to all methods nested inside. This attribute is intentionally not permitted on assemblies. To apply the attribute to multiple type declarations, use it at the module level instead.
```